### DISCUSSION TITLE
GitHub review notification view is context free

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#63030] [Mobile] [answered]
I got a notification for an approval and it's context free:
![Screenshot_20230807-100338.png](https://github.com/community/community/assets/2119212/5efa0a20-1363-4946-815f-48660cd7e07b)

I'd like to at least see the title of the PR and its number as well as the organization and repository. 

If I go back, I get to a screen that has this information, but…

**Comment by mattruggio** [comment, marked as answer]
Thanks for the feedback @jsoref! This is definitely a screen we can improve. I am going to get this UI/UX suggestion filed for our Android engineers to take a look at. In the meantime, can you please confirm this is for our Android app and also let us know what specific application version you are using?

**Reply by jsoref** [reply]
Yes, GitHub for Android version 1.125.0
